Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8
+
+WORKDIR /app
+
+COPY . .
+
+RUN  pip install pipenv
+
+RUN pipenv install fastapi jwt
```

Comment:
Libralies shold be install by `pip` in Docker container, without using pipenv.
I recommend to use `requirements.txt` for definition of dependencies.

Reply:
i recommend adding newline.

Reply:
```
pip install -r requirements.txt
```